src/components/List/index.js

```diff
@@ -0,0 +1,359 @@
+import PropTypes from 'prop-types'
+import React, {
+  forwardRef,
+  useCallback,
+  useContext,
+  useImperativeHandle,
+  useMemo,
+  useRef,
+  useState,
+} from 'react'
+import orderBy from '../../utils/orderBy'
+import ActivityIndicator from '../ActivityIndicator'
+import Box from '../Box'
+import Typography from '../Typography'
+import Cell from './Cell'
+import SelectBar from './SelectBar'
+import ListContext from './context'
+import * as variantExplorer from './variantExplorer'
+import * as variantProduct from './variantProduct'
+import * as variantTable from './variantTable'
+
+const variants = {
+  explorer: variantExplorer,
+  product: variantProduct,
+  table: variantTable,
+}
+
+const Body = ({ children, emptyListText, ...props }) => {
+  const { data, idKey, setRowState, rowsState, isLoading } =
+    useContext(ListContext)
+  if (isLoading === true) {
+    return (
+      <Box display="flex" my={2} justifyContent="center" {...props}>
+        <ActivityIndicator size={50} />
```

Comment:
```suggestion
        <ActivityIndicator size={50} active />
```

Reply:
Would it be possible for a List user to change the Loader to default?

Reply:
Yes, I'll do